content/management/configuration-api/v3.3/index.mdx

```diff
@@ -1267,6 +1280,18 @@ curl -X POST \
 
 <CodeResponse type="configuration" version="v3.3" json="listBots"/>
 
+```json
```

Comment:
This whole example should be removed:
![image](https://user-images.githubusercontent.com/42515529/103542854-e6909880-4e9d-11eb-8014-b9fa885146c4.png)
You need to update the payload in `payloads/management/v3.3/configuration-api/responses/bots/listBots.json`